NashTeamAlpha/BangazonUserAuth
# Added userId to old database as well as new database

## Pull Request Description

The new database was updated to include CustomerId column 
When a user registers for the website a new user is added to the old database and the CustomerId from the old database is grabbed and added to the new database for reference

## Steps to Test

1. `git fetch`
1. `git checkout ddw-registration`
1. Register a new user and make sure that user was added to your old database and the correct CustomerId was added to the new database

## Issue

#2, #3
